docs: document reconstruction visualization (#98)
tier4/CARET_doc: docs/visualization/visualization_api/response_time.md

plot.show()
```

> Note: The `output_notebook()` is needed only when running the script on Jupyter Notebook.

Reviewer: ```suggestion
Note: The `output_notebook()` is needed to show figures inside Jupyter.
```

Author: I modified all sentence as your comment.
https://github.com/tier4/CARET_doc/pull/98/commits/852b5e04f782cfa82109af7d22f6ead698c6b51a
